docs/coding.md

```diff
@@ -0,0 +1,78 @@
+# Coding Conventions
+
+Please follow coding conventions and guidelines described in the following documents:
+
+* [Go proverbs](https://go-proverbs.github.io/) - highly recommended read
+* [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments)
+* [Effective Go](https://golang.org/doc/effective_go.html)
+* [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
+
+Here's a list of some more specific conventions that are often followed in
+the code and will be pointed out in the review process:
+
+## General
+
+* Keep variable names short for variables that are local to the function.
+* Do not export a function or variable name outside the package until you
+  have an external consumer for it.
+* Have setter or getter interfaces/methods to access/manipulate information in
```

Comment:
I had to re-read this sentence four or five time before I understood what the recommendation was. Can we word this more clearly?

Reply:
Removed it :)